src/VirtualClient/VirtualClient.Actions/DiskWorkloadExecutor.cs

```diff
@@ -444,6 +447,39 @@ protected virtual string GetTestFile(string mountPoint)
             return this.PlatformSpecifics.Combine(mountPoint, this.FileName ?? Path.GetRandomFileName());
         }
 
+        /// <summary>
+        /// Returns the name of the test files seperated by whitespace given a mount point.
+        /// </summary>
+        /// <param name="mountPoint">A mount point to the disk under test.</param>
+        /// <returns>
+        /// The full path to the test files seperated by whitespace.
+        /// </returns>
+        protected virtual string GetTestFiles(string mountPoint)
+        {
+            mountPoint.ThrowIfNullOrWhiteSpace(nameof(mountPoint));
+
+            List<string> testFileNames = new List<string>();
+
+            if (!string.IsNullOrWhiteSpace(this.FileName))
+            {
+                testFileNames.AddRange(Regex.Split(this.FileName, $@"\s*{DiskWorkloadExecutor.FileNameParameterDelimiter}\s*").Select(f => f.Trim())
```

Comment:
personal preference: do we need a Regex? or just plain split by comma would work?

Reply:
Regex helps in taking care of the whitespaces if any.

Reply:
I mean you could split by comma and then trim

Reply:
Yes! That's right! We can get rid of the Regex.